src/main/java/io/cdap/plugin/gcp/gcs/sink/GCSBatchSink.java

```diff
@@ -493,8 +485,8 @@ public String getPath() {
     }
 
     @Override
-    public FileFormat getFormat() {
-      return FileFormat.from(format, FileFormat::canWrite);
+    public String getFormatName() {
```

Comment:
How is it going to be used?

Reply:
this gets used by https://github.com/cdapio/hydrator-plugins/blob/19a6d06a538714a5fdcde15eac9084b5bfe2fbba/format-common/src/main/java/io/cdap/plugin/format/plugin/AbstractFileSource.java#L109, it just asks CDAP for a plugin with this name